h_transport_materials/property_database/lipb/lipb.py

```diff
@@ -368,6 +368,25 @@ def atom_density_lipb(nb_li: int, nb_pb: int):
     source="okada_analysis_2012",
 )
 
+
+data_okuno = htm.structure_data_from_wpd("okuno_1986/data.csv")
+
+okuno_diffusivity_li7pb2 = Diffusivity(
+    data_T=1000 / (data_okuno["li7pb2"]["x"] * u.K**-1),
```

Comment:
'x' and 'y' should be capitalized like how it is in the csv file (lines 375, 376, 383, 384)

Reply:
The function `htm.structure_data_from_wpd` restructures the data from the csv file in a more convenient way.
See the documentation of the function:

https://github.com/RemDelaporteMathurin/h-transport-materials/blob/27c53f2cbf39c46a27d8777b13b00b8e7e53dba7/h_transport_materials/helpers.py#L23-L33